code/modules/mob/living/simple_animal/abnormality/zayin/pile_of_mail.dm

```diff
@@ -0,0 +1,303 @@
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail"
+	layer = BELOW_MOB_LAYER
+	duration = 10 SECONDS
+
+//x2 workspeed buff pro 2022 free hack free robux
+/datum/status_effect/workspeed_buff
+	id = "workspeed_buff"
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/workspeed_buff
+	duration = 6 SECONDS
+
+/datum/status_effect/workspeed_buff/on_apply()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	user.physiology.work_speed_mod *= 1.5
+
+/datum/status_effect/workspeed_buff/on_remove()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	user.physiology.work_speed_mod /= 1.5
+
+/atom/movable/screen/alert/status_effect/workspeed_buff
+	name = "A letter from the past"
+	desc = "Reading the letter made you want to try harder for your past friends."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail"
+
+/datum/status_effect/nofear_buff
+	id = "nofear_buff"
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/nofear_buff
+	duration = 6 SECONDS
+
+/datum/status_effect/nofear_buff/on_apply()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	ADD_TRAIT(user, TRAIT_WORKFEAR_IMMUNE, TRAIT_GENERIC)
+	ADD_TRAIT(user, TRAIT_COMBATFEAR_IMMUNE, TRAIT_GENERIC)
+
+/datum/status_effect/nofear_buff/on_remove()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	REMOVE_TRAIT(user,TRAIT_WORKFEAR_IMMUNE,TRAIT_GENERIC)
+	REMOVE_TRAIT(user,TRAIT_COMBATFEAR_IMMUNE,TRAIT_GENERIC)
+
+/atom/movable/screen/alert/status_effect/nofear_buff
+	name = "A letter from the present"
+	desc = "Reading the letter made you fearless."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail"
+
+//yes this is ripped off of paper and mermaid gift but fuck you.
+/obj/item/parcelself
+	name = "Parcel from your alternate self"
+	desc = "You should not see this."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "parcel"
+	inhand_icon_state = "parcel"
+	worn_icon_state = "parcel"
+	drop_sound = 'sound/items/handling/paper_drop.ogg'
+	pickup_sound =  'sound/items/handling/paper_pickup.ogg'
+	gender = NEUTER
+	color = "white"
+	var/mob/living/carbon/receiver = null
+
+//this is using the parcel. you either go insane or get special ego
+/obj/item/parcelself/attack_self(mob/living/carbon/human/user)
+	if(receiver != user)
+		to_chat(user,"<span class='notice'>It feels wrong to try to open someone else's package.</span>")
+		return
+	if(prob(50))
+		new /obj/item/ego_weapon/mail_satchel(get_turf(user))
+		to_chat(user,"<span class='nicegood'>As you unravel the parcel a satchel falls out.</span>")
+	else
+		new /obj/item/clothing/suit/armor/ego_gear/letter_opener(get_turf(user))
+		to_chat(user,"<span class='nicegood'>As you unravel the parcel an outfit falls out.</span>")
+	qdel(src)
+	return
+
+/obj/item/parcelself/New(loc,mob/living/carbon/human/user, ...)
+	. = ..()
+	desc = "This parcel seems to be addressed to [user.name]. Where did it come from..?"
```

Comment:
```suggestion
	desc = "This parcel seems to be addressed to \a [user]. Where did it come from..?"
```
If the user is named something weird through adminbus shenanigans or something else, this will handle it properly. For example, with the name `stranger`, it'd do `This parcel seems to be addressed to a stranger.`, while with the name `Catt` it's do `This parcel seems to be addressed to Catt.`